src/item/GlassBottle.php

```diff
@@ -23,6 +23,32 @@
 
 namespace pocketmine\item;
 
+use pocketmine\block\Block;
+use pocketmine\block\BlockLegacyIds;
+use pocketmine\event\player\PlayerDropItemEvent;
+use pocketmine\math\Vector3;
+use pocketmine\player\Player;
+
 class GlassBottle extends Item{
 
+	public function onInteractBlock(Player $player, Block $blockReplace, Block $blockClicked, int $face, Vector3 $clickVector) : ItemUseResult{
+		if($blockClicked->getId() === BlockLegacyIds::FLOWING_WATER){
```

Comment:
It may not be flowing water. Water most commonly exists in its still form in vanilla.